src/pages/library/DataBrowserToggler.js

```diff
@@ -19,18 +19,23 @@ export const DataBrowserPreviewToggler = ({ onChange, catalogShowing }) => {
   }, [
     div({ style: { display: 'flex', flexDirection: 'column' } }, [
       strong(['Preview the new Data Catalog']),
-      label({
-        role: 'link',
-        style: { fontWeight: 'bold', display: 'flex', flexDirection: 'row', alignItems: 'center', marginTop: 6 }
-      }, [
-        h(Switch, {
-          checked: catalogShowing,
-          onLabel: '', offLabel: '',
-          width: 55, height: 25,
-          onChange
-        }),
-        div({ style: { marginLeft: 10 } }, [`New Catalog ${catalogShowing ? 'ON' : 'OFF'}`])
-      ])
+      h(IdContainer, [id => h(Fragment, [
+        div({
+          style: { display: 'flex', flexDirection: 'row', alignItems: 'center', marginTop: 6 }
+        }, [
+          h(Switch, {
+            id,
+            role: 'switch',
+            'aria-checked': catalogShowing,
```

Comment:
Does the Switch component not set role and aria-checked itself?

Reply:
Surprisingly, they don’t! I was thinking of plumbing them into the root component, would you like me to?

Reply:
If I remove these lines, the role and aria-checked attributes are still there.
![Screen Shot 2022-11-09 at 10 17 08 AM](https://user-images.githubusercontent.com/1156625/200868876-a86f636b-aa9c-44c4-8b6d-8fcfd406f8d2.png)

[react-switch](https://www.npmjs.com/package/react-switch)'s demo also includes those attributes.

Reply:
You're right! I wasn't going far enough down the component chain.